src/components/home/Home.js

```diff
@@ -7,83 +7,63 @@ import {SORT_TYPES} from "../../constants/Constant";
 import SortOptions from "../sort/SortOptions";
 import PropTypes from "prop-types";
 
-class Home extends Component {
-  state = {
-    movies: this.props.movies,
-    currentGenre: "All",
-    isAskOrder: true
-  }
+function Home(props) {
+  const [movies, setMovies] = useState(props.movies);
```

Comment:
Фильмы по умолчанию должны быть отсортированны, не вижу стейт в котором хранится поле по которому сортируется массив, вижу только направление.
```suggestion
  const [movies, setMovies] = useState(props.movies);
  const [sort, setSort] = useState({ field: 'title', order: 'asc' }) // например
```

Reply:
Done